### PR TITLE
minor go tutorial tweaks

### PR DESCRIPTION
**Pull request by sw-yx**
## What does this PR do?

minor go tutorial tweaks to make things a little faster

### CODE REVIEW

File: docs/go/introduction.md [hunk 1]
```diff
@@ -6,14 +6,31 @@ sidebar_label: Go SDK introduction
...
+You can now view Temporal Web at http://localhost:8088.
+
+3. Run the worker and starter included in the project.
+
+```bash
+go run worker/main.go
+go run start/main.go
+```
+
+If you have [`nodemon`](https://nodemon.io/) installed, you can automatically reload when you change any files: `nodemon --watch './**/*.go' --signal SIGTERM --exec 'go' run worker/main.go`
```
Comment (flossypurse) [inline docs/go/introduction.md:32]: This type of info seems like general "developer tip" type of info that is not specific to Temporal, but just general developer tooling.

I am definitely not opposed to having this through out the docs, but it would be ideal if it existed in a special container that identified it as "general developer tips".

File: docs/go/introduction.md [hunk 2]
```diff
@@ -6,14 +6,31 @@ sidebar_label: Go SDK introduction
...
-```
-go get go.temporal.io/sdk@latest
+```bash
+git clone https://github.com/temporalio/hello-world-project-template-go
 ```
 
 2. [Install and run the Temporal Server](/docs/server/quick-install) using `docker compose`.
 
+```bash
+git clone https://github.com/temporalio/docker-compose.git
```
Comment (flossypurse) [inline docs/go/introduction.md:18]: Can we try to use the quick-install component being added here: https://github.com/temporalio/documentation/pull/548 ?

File: docs/go/tutorial-prerequisites.md [hunk 3]
```diff
@@ -12,4 +12,14 @@ Make sure you have [Go](https://golang.org/doc/install) installed. These tutoria
 
 ## Temporal server
 
-Download, install, and run the Temporal server via `docker-compose`. You can find Temporal server setup instructions here: [https://docs.temporal.io/docs/server/quick-install](/docs/server/quick-install). It is easy to do and you can keep it running in the background while you build applications.
+Download, install, and run the Temporal server via `docker-compose`.
```
Comment (flossypurse) [inline docs/go/tutorial-prerequisites.md:15]: Same here, lets try using the reusable content page?

File: docs/go/introduction.md (hunk 2 above)
Comment (sw-yx) [reply docs/go/introduction.md:18]: i had a look but its a little too wordy for the style of this tutorial tbh

**Review by flossypurse — APPROVED** [review]
